README.rst

```diff
@@ -135,7 +139,8 @@ Contributing
 History
 -------
 
-Recent changes can be viewed in the `CHANGELOG.md <CHANGELOG.md>`__
+Recent changes can be viewed in the `CHANGES.rst
+<https://github.com/pauperpythonistas/python-geomark/blob/master/CHANGES.rst>`__
```

Comment:
The url here is giving 404s.

Reply:
That's because this file doesn't exist on master yet.  Master currently has CHANGELOG.md  after this is merged, this link will work.

Reply:
mmm pretty sure it's there already:
https://github.com/pauperpythonistas/python-geomark/blob/master/CHANGELOG.rst

Reply:
Oh, you have a typo in the filename. You have `CHANGES.rst`

Reply:
As I was browsing around for tools to preview rst files and searching for python/pip conventions, it came to my attention that the file is usually called CHANGES.rst, not CHANGELOG.rst, so I changed the name all whimsical like seeing as it was currently empty, and I created it to start with.

Reply:
right-o.